src/content/docs/alerts-applied-intelligence/new-relic-alerts/alerts-nerdgraph/nerdgraph-api-notification-channels.mdx

```diff
@@ -3,367 +3,689 @@ title: "NerdGraph tutorial: Notification channels"
+                    apiKey
+                    dataCenterRegion
+                    recipients
+                    tags
+                    teams
+                  }
+                }
+                ... on AlertsHipChatNotificationChannel {
+                  config {
+                    authToken
+                    baseUrl
+                    roomId
+                  }
+                }
+                ... on AlertsEmailNotificationChannel {
+                  config {
+                    emails
+                    includeJson
+                  }
+                }
+                ... on AlertsCampfireNotificationChannel {
+                  config {
+                    room
+                    subdomain
+                    token
+                  }
+                }
+              }
+              totalCount
+              nextCursor
+            }
+          }
+        }
+      }
+    }
+    ```
+  </Collapser>
+
+  <Collapser
+    id="paginate"
+    title="Paginate through notification channels with cursor pagination"
   >
+    If a given account's list of notification channels exceeds the 200 channel page limit, you can use the pagination cursor to retrieve additional pages.
+
+    With cursor pagination, you continue to request additional pages using the `nextCursor` until that field returns empty in the response. An empty `nextCursor` signals that you have reached the end of the result set.
+
+    Here is an example:
+
     ```
     {
       actor {
-        account(id: $accountId) {
+        account(id: <var>YOUR_ACCOUNT_ID</var>) {
           alerts {
-            notificationChannels(cursor: $cursor) {
-              # selection set here
-    } } } } }
+            notificationChannels {
+              channels {
+                id
+                name
+                type
+              }
+              totalCount
+              nextCursor
+            }
+          }
+        }
+      }
+    }
     ```
-    Type for results:
+
+    The code above returns a set of results like this:
+
+    ```
+    {
+      "data": {
+        "actor": {
+          "account": {
+            "alerts": {
+              "notificationChannels": {
+                "channels": [
+                  {
+                    "id": "250",
+                    "name": "Channel 1",
+                    "type": "SLACK"
+                  },
+                  {
+                    "id": "713",
+                    "name": "Channel 2",
+                    "type": "WEBHOOK"
+                  },
+                  // ... +198 more notification channels in reality
+                ],
+                "nextCursor": "Wh4LK9JYzfACVlNkyvf7Rg==:I5VbSEpgx3UWNA5AOVsUPv4=",
+                "totalCount": 268
+              }
+            }
+          }
+        }
+      }
+    }
+    ```
+
+    In your next request, provide the cursor like so, updating each subsequent request to return the updated cursor, until the cursor is empty:
```

Comment:
I don't follow this sentence, what do you mean?

Reply:
It borrows from the existing explanation of how the GraphQL cursors work -- borrowed from the docs about [Alerts policies in NerdGraph](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alerts-nerdgraph/nerdgraph-api-alerts-policies/). Each page of results will return a unique cursor string for retrieving the next page of results. When you've reached the last page, you'll get back a `null` for the empty cursor.